cas-server-documentation/integration/Delegate-Authentication.md

```diff
@@ -153,7 +87,7 @@ To add SAML support, add the following dependency:
 
 A provider is a server which can authenticate user (like Google, Yahoo...) instead of a CAS server. If you want to delegate the CAS authentication to Twitter for example, you have to add an OAuth client for the provider: Twitter. Clients classes are defined in the pac4j library.
 
-All the needed clients to authenticate against providers must be declared in the `applicationContext.xml` file:
+All the needed clients to authenticate against providers must be declared in a specific `WEB-INF/spring-configuration/pac4jContext.xml` file:
 
 {% highlight xml %}
 <bean id="facebook1" class="org.pac4j.oauth.client.FacebookClient">
```

Comment:
I really wouldnt want anyone to start configuring XML unless they absolutely have to :))